internal/ethapi/api.go

```diff
@@ -1405,7 +1405,7 @@ func AccessList(ctx context.Context, b Backend, blockNrOrHash rpc.BlockNumberOrH
 	}
 	isPostMerge := header.Difficulty.Cmp(common.Big0) == 0
 	// Retrieve the precompiles since they don't need to be added to the access list
-	precompiles := vm.ActivePrecompiles(b.ChainConfig().Rules(header.Number, isPostMerge))
+	precompiles := vm.DefaultActivePrecompiles(b.ChainConfig().Rules(header.Number, isPostMerge))
```

Comment:
[Optional] If we use `vmenv.ActivePrecompiles` instead of vm.`DefaultActivePrecompiles` in go-ethereum/core/vm/runtime/runtime.go, I think same change should be applied to this part.

Reply:
Reverted changes to runtime.go. 
Execute, Create, Call functions are performed via evm binary, which is the evm executor. 
The point at which the precompiled contract address is added to the active precompiles list is when ethermint executes an evm tx, so there is no need to change to ActivePrecompiles in parts that do not have this access route.